qri/dataset.py

```diff
@@ -125,7 +148,30 @@ def __repr__(self):
         return 'Dataset("%s")' % self.human_ref()
 
     def _repr_html_(self):
-        return '<code>Dataset("%s")</code>' % self.human_ref()
+        title = self.human_ref()
+        meta = self.meta_component
+        if meta is None:
+            meta = Meta({
+                "title": "untitled dataset",
```

Comment:
For consistency, use `'` single quotes instead of `"` double quotes in this file.